interactions/client/context.py

```diff
@@ -199,16 +199,15 @@ async def defer(
                 application_id=int(self.id),
                 data={"type": self.callback.value, "data": {"flags": is_ephemeral}},
             )
-
             with suppress(LibraryException):
                 res = await self._client.get_original_interaction_response(
                     self.token, str(self.application_id)
                 )
-                self.message = Message(**res, _client=self._client)
+                _message = Message(**res, _client=self._client)
```

Comment:
It's should not be here

Reply:
Hmm somehow my branching got problem I'll fix that